src/test/java/io/skai/okta/internshipstreamingserviceparserxml/services/converter/EpisodeAdditionalInfoConverterTest.java

```diff
@@ -0,0 +1,54 @@
+package io.skai.okta.internshipstreamingserviceparserxml.services.converter;
+
+import io.skai.okta.internshipstreamingserviceparserxml.dto.EpisodeAdditionalInfo;
+import io.skai.okta.internshipstreamingserviceparserxml.dto.ImdbJsonItem;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class EpisodeAdditionalInfoConverterTest {
+    private EpisodeAdditionalInfo additionalInfo;
+    private static final Pattern ID_PATTERN = Pattern.compile("(?<=t{2})\\d+");
+
+    @BeforeEach
+    void setUp() {
+        EpisodeAdditionalInfoConverter converter = new EpisodeAdditionalInfoConverter();
+
+        ImdbJsonItem item = ImdbJsonItem.builder()
+                                        .imdbID("tt1480055")
+                                        .seriesID("tt0944947")
+                                        .imdbRating("9.1")
+                                        .imdbVotes("44868")
+                                        .runtime("62 min")
+                                        .build();
+
+        additionalInfo = converter.convert(9379992, item);
```

Comment:
you should run convert in test instead of setUp method, before hook used to init some stuff